pandas/core/internals/array_manager.py

```diff
@@ -90,10 +90,10 @@
     from pandas import Float64Index
 
 
-T = TypeVar("T", bound="ArrayManager")
+T = TypeVar("T", bound="BaseArrayManager")
```

Comment:
Good afternoon. Why not give it a more expressive name?

Reply:
bc its really only for mypy